### PR TITLE
ci: release job iteration

### PR DESCRIPTION
**Pull request by armcknight**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
       - master
+      
+  pull_request:
+    paths:
+      - '.github/workflows/release.yml'
```
Comment (realkosty) [inline .github/workflows/release.yml:9]: This is temporary while testing, right?

Comment (armcknight) [reply .github/workflows/release.yml:9]: Yep! We can remove it now that it's working.

Comment (armcknight) [inline .github/workflows/release.yml:9]:
```suggestion
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
-      - master
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version to release
+        required: true
+      
+  pull_request:
+    paths:
+      - '.github/workflows/release.yml'
```
Comment (armcknight) [inline .github/workflows/release.yml:11]:
```suggestion
```

**Pull request merged**